netbox/dcim/choices.py

```diff
@@ -278,6 +278,15 @@ class PowerPortTypeChoices(ChoiceSet):
     TYPE_ITA_M = 'ita-m'
     TYPE_ITA_N = 'ita-n'
     TYPE_ITA_O = 'ita-o'
+    # USB
```

Comment:
Minor nit, but can we keep these in the same order as `ConsolePortTypeChoices`? Or update that choice set to match this one? I don't have a preference for the order, just want them to be consistent.

Reply:
Yeah no problem, i'll change that now